Libraries/Lists/VirtualizedList.js

```diff
@@ -1046,7 +1111,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     }
   };
 
-  _onCellLayout(e, cellKey, index) {
+  _onCellLayout = (e, cellKey, index): void => {
     const layout = e.nativeEvent.layout;
```

Comment:
```
if (isNaN(index)) {
  index = 0
}
```
Maybe you should check index before actually use it. 
```
this._highestMeasuredFrameIndex = Math.max(
         this._highestMeasuredFrameIndex,
         index,	 
       );	
  ```
If index is NaN, then _highestMeasuredFrameIndex will be also NaN, which may leads to crash.